app/models/reducer.rb

```diff
@@ -1,5 +1,6 @@
 class Reducer < ApplicationRecord
   include Configurable
+  include Reducible
```

Comment:
`Reducible` is perhaps the wrong name for the module. Including feels like a "is-a" relation. Reducer isn't a Reducible, workflow is. Maybe just rename it to `BelongsToReducible`?

Reply:
Sure. This is basically a temporary situation but the less confusion, the better. That's why I pulled this out into a concern in the first place.